administrator/components/com_admin/sql/updates/postgresql/4.2.0-2022-06-16.sql

```diff
@@ -1,55 +1,56 @@
 --
--- Table structure for table "#__guidedtours"
+-- Table structure for table `#__guidedtours`
 --
 --
 CREATE TABLE IF NOT EXISTS "#__guidedtours"
 (
- `id` int NOT NULL AUTO_INCREMENT,
-  `asset_id` int DEFAULT 0,
-  `title` varchar(255) NOT NULL,
-  `description` text NOT NULL,
-  `ordering` int NOT NULL DEFAULT 0,
-  `extensions` text NOT NULL,
-  `url` varchar(255) NOT NULL,
-  `overlay` tinyint NOT NULL  DEFAULT 0,
-  `created` datetime NOT NULL,
-  `created_by` int NOT NULL DEFAULT 0,
-  `modified` datetime NOT NULL,
-  `modified_by` int NOT NULL DEFAULT 0,
-  `checked_out_time` datetime NOT NULL,
-  `checked_out` int NOT NULL DEFAULT 0,
-  `published` tinyint NOT NULL DEFAULT 0,
-  `state` tinyint NOT NULL DEFAULT '1',
-    PRIMARY KEY ("id")
+  "id" int NOT NULL AUTO_INCREMENT,
```

Comment:
this for postgresql is wrong
should be
`"id" serial NOT NULL,`

there is no unsigned  in postgres

Reply:
Thanks, @alikon for reviewing the PR. It has been taken care of in PR #10 by @richard67.